src/client/builtin_scene/materials/sdf_text.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include "../material_base.hpp"
+#include <memory>
+#include <glm/glm.hpp>
+
+namespace builtin_scene::materials
+{
+  /**
+   * Material for rendering SDF-based text
+   */
+  class SDFTextMaterial : public Material
```

Comment:
We don't need a new material for SDF text rendering, it should be included in WebContentMaterial.